boards/native/native_sim/doc/index.rst

```diff
@@ -59,6 +59,32 @@ To build, simply specify the ``native_sim`` board as target:
    :goals: build
    :compact:
 
+Cross Compiling
+===============
+
+To cross-compile for the ``native_sim`` board (for example, targeting aarch64), use the following command:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :host-os: unix
+   :board: native_sim/native/64
+   :goals: build
+   :cmake-args: -DZEPHYR_TOOLCHAIN_VARIANT=cross-compile -DZEPHYR_POSIX_ARCH=aarch64 -DCROSS_COMPILE=<path-to-cross-compiler>/aarch64-none-linux-gnu- -DSYSROOT_DIR=<path-to-sysroot>
```

Comment:
Typically we provide instructions with all steps so people can get things running from a plain Ubuntu install. Setting things for cross compiling is hardly as trivial as for building for ones' host. But it would be much nicer if we could provide instructions for a reasonable setup. Otherwise in no time I expect bug reports due to not being able to install the compiler, and/or not being able to make it work, or asking for instructions.